### ISSUE TITLE
Can we ID customer on phone in phone conversation 

### ISSUE THREAD

**Issue by sarsony1** [freescout-helpdesk/freescout#3786]
Hello team, 
I see a new customer is created everytime in a phone conversation even when the phone number is same. Similar to email, can we id customer based on phone number when it comes to phone conversation. It kind of make sense to me. Let me know what you think

**Comment by freescout-helpdesk** [comment]
Are you creating the phone conversation via web interface?

When you are entering "Phone" in the new phone conversation FreeScout shows you the list of customers with this phone number.

**Comment by sarsony1** [comment]
No. I am using the API and everytime it creates a new customer.

**Comment by freescout-helpdesk** [comment]
Please provide your API request.

**Comment by freescout-helpdesk** [comment]
Fixed in API & Webhooks Module v1.0.75
